sympy/calculus/finite_diff.py

```diff
@@ -46,45 +50,63 @@ def finite_diff_weights(order, x_list, x0):
     list
         A list of sublists, each corresponding to coefficients for
         increasing derivative order, and each containing lists of
-        coefficients for increasing accuracy.
+        coefficients for increasing subsets of x_list.
 
     Examples
     ========
 
     >>> from sympy import S
     >>> from sympy.calculus import finite_diff_weights
-    >>> finite_diff_weights(1, [-S(1)/2, S(1)/2, S(3)/2, S(5)/2], 0)
+    >>> res = finite_diff_weights(1, [-S(1)/2, S(1)/2, S(3)/2, S(5)/2], 0)
```

Comment:
I think you will need to add a line `>>> res` to make doctests pass here, we will see.